index.html

```diff
@@ -57,16 +70,6 @@ <h3>Commands</h3>
   </div>
 </div>
 
-<div class="p-strip--accent is-bordered is-shallow">
-  <div class="row">
-    <div class="col-8">
-      <h3>Commercial support</h3>
-      <p>For commerical support please contact us</p>
-      <p><a href="/contact-us" class="p-button">Contact support</a></p>
-    </div>
-  </div>
-</div>
-
```

Comment:
No commercial support?

Reply:
I think we need to get help from Thibaut first.